<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>game</class>
 <widget class="QWidget" name="game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>460</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>460</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>460</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="DownButton">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>380</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>↓</string>
   </property>
  </widget>
  <widget class="QPushButton" name="UpButton">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>220</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>↑</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RightButton">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>300</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>→</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LeftButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>300</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>←</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ScorellineEdit">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>10</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>    0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LevellineEdit">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>90</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>    1</string>
   </property>
  </widget>
  <widget class="QLabel" name="Scorelabel">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>10</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;Score：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="Levellabel">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>90</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;Level：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BackButton">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>420</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
